bundles/GraphicServiceOrder/Service/OrderService.php

```diff
@@ -324,6 +335,7 @@ private function createOrderTask(GsOrder $gsOrder)
                 ],
                 $this->hammerService->getCustomFieldId('Debitor') => (string) $gsOrder->getDebitor(),
                 $this->hammerService->getCustomFieldId('Marketing Account') => $gsOrder->getMarketingAccount() ? [0 => ['value' => 'Markedsføringskonto']] : null,
+                $this->hammerService->getCustomFieldId('Delivery Note URL') => $this->requestStack->getCurrentRequest()->getSchemeAndHttpHost().'/jira/delivery_note/'.$gsOrder->getId(),
```

Comment:
You should use `UrlGeneratorInterface` for generating urls. See https://symfony.com/doc/master/routing.html#generating-urls-in-services for an example.

Reply:
Changed